### PR TITLE
fix: unable to terminate a compute session when wsproxy address is not properly

### PR DESCRIPTION
**Pull request by pderer**
There was a bug that the session was not deleted when the wsproxy address of the scaling group was invalid.

This PR is related to this PR in backend.ai repository. [#1423](https://github.com/lablup/backend.ai/pull/1423)

In order to prevent side effects caused by server-side code modification, it was modified to handle exceptions in the web UI-side code.

### CODE REVIEW

File: src/components/backend-ai-session-list.ts [hunk 1]
```diff
@@ -1462,35 +1462,47 @@ export default class BackendAISessionList extends BackendAIPage {
...
-        if ('description' in err) {
-          this.notification.text = PainKiller.relieve(err.description);
-        } else {
-          this.notification.text = PainKiller.relieve('Problem occurred during termination.');
-        }
-        this.notification.show(true, err);
-      });
+      this._requestDestroySession(sessionId, accessKey, forced);
     }).catch((err) => {
       console.log(err);
```
Comment (fregataa) [inline src/components/backend-ai-session-list.ts]: could you remove this console.log?
it is not written in this pr but it would be good!

**Review by fregataa — COMMENTED** [review]
Thanks! it seems good for me.
Please wait for other's review!

File: src/components/backend-ai-session-list.ts (hunk 1 above)
Comment (pderer) [reply src/components/backend-ai-session-list.ts]: Thanks for reviewing!
I will fix it.

**Review by inureyes — APPROVED** [review]
LGTM

**Pull request merged**